### DISCUSSION TITLE
Not Working For Magic Mouse

### DISCUSSION THREAD

**Discussion by lalalalatte** [noah-nuebling/mac-mouse-fix#457] [Questions] [answered]
Basically title, any adjustments I make don't appear on the mouse. I've enabled permissions for it on system settings. The same problem happened with Mos (I tried it because this one didn't work), but both of them aren't working. If its relevant I'm using a magic mouse. Using Ventura 13.0.1.

**Comment by noah-nuebling** [comment, marked as answer]
Hey @lalalalatte, thanks for reaching out!

Mac Mouse Fix and MOS don't support the Magic Mouse. I might add Magic Mouse support in the future but it's not currently planned. 

I think BetterTouchTool should work better for you:

https://alternativeto.net/software/bettertouchtool/?platform=mac

Hope that helps! Let me know if you have more questions.

**Reply by lalalalatte** [reply]
Thanks for replying! I also found another application that works for me. https://pilotmoon.com/scrollreverser/
